### PR TITLE
Add prometheus-arkouda-exporter docker image

### LINKED ISSUE #85: Add prometheus-arkouda-exporter Docker image
Add a prometheus-arkouda-exporter image that enables deployment of th [arkouda_metrics_exporter](https://github.com/Bears-R-Us/arkouda-contrib/tree/main/arkouda_metrics_exporter) prometheus exporter to Kubernetes and Docker compose

### PR DESCRIPTION
**Pull request by hokiegeek2**
Closes #85 

This PR adds a prometheus-arkouda-exporter Docker image and build instructions

### CODE REVIEW

File: arkouda-docker/README.md [hunk 1]
```diff
@@ -208,4 +245,9 @@ python build_docker_image.py --chapel_version=1.29.0 --image_type=chapel-gasnet-
 python build_docker_image.py --arkouda_tag=v2023.02.08 --chapel_version=1.29.0 --image_type=arkouda-udp-server
 ```
 
+### prometheus-arkouda-exporter
+
+```
+python build_docker_image.py --arkouda_tag=v2023.02.08 --image_type=prometheus-arkouda-exporter
```
Comment (Ethan-DeBandi99) [inline arkouda-docker/README.md:251]: Just curious here, should we list a later tag. I know in the past we have built the images with a specific tag. Might be worth some discussion about how to keep these up to date without significantly increasing workload.

**Review by Ethan-DeBandi99 — APPROVED** [review]
Looks fine to me. Just on question as far as keeping images up-to-date with Arkouda releases, but I think that applies to more than just this PR.

**Review by jaketrookman — APPROVED** [review]
Looks good

File: arkouda-docker/README.md (hunk 1 above)
Comment (hokiegeek2) [reply arkouda-docker/README.md:251]: ah, good point, I'll update the README.md. I'll specicy these are example build commands and that the arkouda tag should be the latest

Comment (hokiegeek2) [reply arkouda-docker/README.md]: @Ethan-DeBandi99 updated the README, please check it out and lemme know what you think.

**Review by pierce314159 — APPROVED** [review]

**Pull request merged**